cli/preprocessor.go

```diff
@@ -32,41 +34,46 @@ func processInput(
 		fmt.Println("done")
 	}
 
-	// Feed the compiler with the input
-	if !silentFlag {
-		fmt.Print("Compiling ... ")
-	}
+	if benchmarkRuns <= 1 { // Normal compiler mode
```

Comment:
For `benchmarkRuns == 1` I'd still expect the benchmark to be run once...

Reply:
Good point. Will fix that ...